local-test/mock_pipecat_cloud.py

```diff
@@ -116,7 +149,107 @@ async def proxy_request(agent_name: str, session_id: str, path: str, request: Re
             )
 
 
+# ---------------- WhatsApp specific routes ----------------
+@app.get(
+    "/whatsapp",
+    summary="Verify WhatsApp webhook",
+    description="Handles WhatsApp webhook verification requests from Meta",
+)
+async def verify_webhook(request: Request):
```

Comment:
@cbrianhill, if I’m not mistaken, as discussed on slack, we need to implement an equivalent endpoint inside the API service (TypeScript), and in this case, the verification token will be the Pipecat public API key.

Reply:
PR adding support to the sandbox: https://github.com/daily-co/pipecat-cloud-sandbox/pull/274